src/apps/components/AppFrame/AppFrame.tsx

```diff
@@ -19,6 +20,8 @@ interface Props {
   className?: string;
   params?: AppDetailsUrlQueryParams;
   refetch?: () => void;
+  dashboardVersion: string;
+  coreVersion?: string;
```

Comment:
**question**: why is dashboard version required, and core optional? I don't see fallback in `postToExtension`

Reply:
For me, all should be required, but dashboard version is static  - from ENV, while core is async - fetched. 

IMO useShop() should block dashboard rendering, but I can't do that because it breaks ~60 tests.

I will revisit this - maybe I will just prevent rendering in Apps context only

Reply:
I checked this. I can't do it other way right now. AppFrame is used by ExternalAppProvider. I can't "early return" there, because it will break tests. This requires large refactor